development/build/scripts.js

```diff
@@ -1181,6 +1207,18 @@ async function setEnvironmentVariables({
       testing,
     }),
     METAMASK_DEBUG: devMode || variables.getMaybe('METAMASK_DEBUG') === true,
+    METAMASK_BUILD_NAME: getBuildName({
+      environment,
+      buildType,
+      applyLavaMoat,
+      shouldIncludeSnow,
+    }),
+    METAMASK_BUILD_APP_ID: getBuildAppId({
+      buildType,
+    }),
+    METAMASK_BUILD_ICON: getBuildIcon({
+      buildType,
+    }),
```

Comment:
I inject these build time values into the env vars so that they can be accessed in `inpage` to initialize the provider. I'm not sure if there is a better approach or not